dist/alt-browser-with-addons.js

```diff
@@ -1353,11 +1349,24 @@ var createStoreFromObject = function (alt, StoreModel, key, saveStore) {
   return storeInstance;
 };
 
+var checkDispatcherInterface = function () {
+  var dispatcher = arguments[0] === undefined ? {} : arguments[0];
+
+  var requiredMethods = ["waitFor", "dispatchToken", "register", "dispatch"];
```

Comment:
That's a good point, I am going to remove this because it will be the user's responsibility to ensure they are using a dispatcher that will work.  No point in having extra code in alt to do that.

I will remove that, and add docs, then this should be good to go.